src/app/state/importaccounts/saga.ts

```diff
@@ -109,7 +147,11 @@ function* enumerateAccountsFromLedger() {
     yield* put(importAccountsActions.operationFailed(payload))
   } finally {
     if (transport) {
-      yield* call([transport, transport.close])
+      try {
+        yield* call([transport, transport.close])
+      } catch (error) {
+        console.log('Error while closing Ledger USB interface:', error)
```

Comment:
Shouldn't we throw  WalletError?

Reply:
No, I think not, because this happens when we are already trying to close things. Most often this is caused by some other event "closing" the connection before we do, like unplugging the USB. But by this time, we have already handled the error in the above code, this is just some cleanup.